src/librustdoc/html/static/js/search.js

```diff
@@ -1314,22 +1315,29 @@ window.initSearch = function(rawSearchIndex) {
             return;
         }
 
+        var filterCrates = searchState.getFilterCrates(params);
```

Comment:
Here you're passing `params` to `getFilterCrates` but it doesn't actually take a parameter, and you handle params below instead.